docs/tutorials/04_stream/viz_interaction.py

```diff
@@ -0,0 +1,175 @@
+"""
+====================================
+Streaming FURY with user interaction
+====================================
+
+In this tutorial, we show how to use the FURY Streaming system to
+serve an interactive visualization through a web browser.
+
+You can choose between two differnet encodings: WebRTC or MJPEG.
+WebRTC is a more robust option and can be used to perform
+a live streaming with a low-latency connection for exampel using 
+ngrok. However, to use webRTC you need to install the aiortc library.
+
+.. code-block:: bash
+    pip install aiortc
+
+In addition, if you don't have ffmpeg installed, you need 
+to install it.
+
+Linux
+
+
+`apt install libavdevice-dev libavfilter-dev libopus-dev libvpx-dev pkg-config`
```

Comment:
I do not see ffmpeg here. Is that okay?

Reply:
it seems that for mac, ffmpeg, opus or libvpx are not needed. I've just pipinstall aiortc